Application/learn_static.py

```
@@ -141,16 +141,19 @@ def evaluate(self, epoch, save = True, logs = {}):
         self.uniquenesses.append(unique)
         set_uniqueness_history(self.uniquenesses)
 
+        pessimism = (self.compare_acc * 0.95)**2
+        #TRex.log(f"uniquenesses: {self.uniquenesses}, pessimistic previous best: {pessimism}, best: {self.best_result['unique']}")
+
         if unique >= acceptable_uniqueness() and self.settings["accumulation_step"] >= -1:
```

Ruff (0.8.2)

Undefined name `acceptable_uniqueness`

(F821)